src/hooks/useErc20.ts

```diff
@@ -32,9 +38,15 @@ async function _fetchErc20FromNetwork(params: {
     // Causing the caller to never know we got the token it was looking for
     return { ...nativeToken, address }
   }
-
+  const cacheKey = `${networkId}-${address}`
+  const cachedData = getCachedData(cacheKey, ERC20_INFO_CACHE)
```

Comment:
`getCachedData` looks redundant. Bellow we just use `ERC20_INFO_CACHE.set()` I think here we also can just `ERC20_INFO_CACHE.get()`